### DISCUSSION TITLE
Is additional rotation on SvgChevronRight in SubRowExpander.tsx file intentional or a bug?

### DISCUSSION THREAD

**Discussion by DenisMicha** [iTwin/iTwinUI#1749] [Q&A]
_SvgChevronRight_ icon in **SubRowExpander.tsx** file has a conditional "transform" property with 90deg rotation, although it is wrapped by IconButton component, which already has the 90deg rotation included, therefore it results in the icon rotating 180deg on rows expansion. Is it intended for the icon there to rotate 180 degrees, or is that a bug?
![image](https://github.com/iTwin/iTwinUI/assets/139454656/e3b22a2e-e8b9-4801-9ac0-354515d99f43)

**Comment by mayank99** [comment]
there should be no rotation on the IconButton, you might have some custom CSS messing up the intended styling.

**Reply by DenisMicha** [reply]
Clarified in this issue: https://github.com/iTwin/iTwinUI/issues/1750
